common/scala/Dockerfile

```diff
@@ -14,8 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u172-b11
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.3_7
```

Comment:
We are using adoptopenjdk/openjdk11:x86_64-debian-jdk-11.0.5_10 in production with no issues.